types/bids.yaml

```diff
@@ -37,3 +37,11 @@ BidTraceV2WithTimestamp:
           $ref: "./misc.yaml#/Int64"
         timestamp_ms:
           $ref: "./misc.yaml#/Int64"
+
+BidTraceV3:
```

Comment:
perhaps would be better to remove the `BidTraceV2WithTimestamp` type and just have `BidTraceV3` and use that?

Reply:
Makes sense